draft-ietf-sframe-enc.md

```diff
@@ -54,47 +54,52 @@ informative:
 
 --- abstract
 
-This document describes the Secure Frame (SFrame) end-to-end encryption and authentication mechanism for media frames in a multiparty conference call, in which central media servers (SFUs) can access the media metadata needed to make forwarding decisions without having access to the actual media.
-The proposed mechanism differs from other approaches through its use of media frames as the encryptable unit, instead of individual RTP packets, which makes it more bandwidth efficient and also allows use with non-RTP transports.
+This document describes the Secure Frame (SFrame) end-to-end encryption and authentication mechanism for media frames in a multiparty conference call, in which central media servers (selective forwarding units or SFUs) can access the media metadata needed to make forwarding decisions without having access to the actual media.
+
+The proposed mechanism differs from the Secure Real-Time Protocol (SRTP) in that
+it is independent of RTP (thus compatible with non-RTP media transport) and can
+be applied to whole media frames in order to be more bandwidth efficient.
 
 --- middle
 
 
 # Introduction
-Modern multi-party video call systems use Selective Forwarding Unit (SFU) servers to efficiently route RTP streams to call endpoints based on factors such as available bandwidth, desired video size, codec support, and other factors. In order for the SFU to work properly though, it needs to be able to access RTP metadata and RTCP feedback messages, which is not possible if all RTP/RTCP traffic is end-to-end encrypted.
```

Comment:
Do we need to spell out SFU since we already clarified what this is in the abstract?

Reply:
Seems like it doesn't hurt, and might improve clarity if someone skips the abstract.